code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -374,6 +374,24 @@ GLOBAL_VAR_INIT(cryo_overlay_cover_off, mutable_appearance('icons/obj/cryogenics
 		if (do_after(user, 2.5 SECONDS, target=target))
 			close_machine(target)
 
+/obj/machinery/atmospherics/components/unary/cryo_cell/screwdriver_act(mob/living/user, obj/item/tool)
+	if(!on && !occupant && !state_open && (default_deconstruction_screwdriver(user, "pod-off", "pod-off", tool)))
+		update_appearance()
+	else
+		to_chat(user, "<span class='warning'>You can't access the maintenance panel while the pod is " \
+		+ (on ? "active" : (occupant ? "full" : "open")) + "!</span>")
+	return TRUE
```

Comment:
I don't know how to fix this but I really wish it didn't exist

Reply:
6 year old cryo code is a work of art